configure.ac

```diff
@@ -1221,6 +1227,14 @@ dnl it would break GCC's #include_next.
 AC_DEFUN([SUPPRESS_WARNINGS],
          [$(echo $1 |${SED} -E -e 's/(^| )-I/\1-isystem /g' -e 's;-isystem /usr/include([/ ]|$);-I/usr/include\1;g')])
 
+AC_MSG_CHECKING([whether test/fuzz/fuzz main function is needed])
+if [[[ $use_sanitizers == *"fuzzer"* ]]]; then
```

Comment:
I am pretty sure this will break honggfuzz. Is there anything wrong with how this was checked previously?

Reply:
Ok, once you rebase I expect honggfuzz to be broken